pset-functions.js

```diff
@@ -135,7 +178,19 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+function scrabbleScore(word){
+scrabbleLegend = {a:1, e:1, i:1, o:1, u:1, l:1, n:1, r:1, s:1, t:1, d:2, g:2, b:3, c:3, m:3, p:3, f:4, h:4, v:4, w:4, y:4, k:5, j:8, x:6, q:10, z:10}
```

Comment:
@cbrooks2019  nice job using an object to represent the letter to score combinations